src/ActionPanel.scss

```diff
@@ -1,57 +1,91 @@
 @import 'variables';
 
-// TODO: Rework the arrow implementation. For example, it is not quite
-//       centered with respect to the action panel.
-
-.ActionPanel__panel {
-    width: $c2lc-action-panel-width;
+.ActionPanel__background {
+    align-self: center;
     height: $c2lc-action-panel-height;
```

Comment:
This div has same height and width as the inner child, which makes setting height and width to this container to have no effect. Making this container to automatically calculate its height and width from inner contents is a good approach here. We can remove width and height.

Reply:
Good to know, and removed.